### PR TITLE
Port goreleaser.yml config

### PR DESCRIPTION
**Pull request by KeisukeYamashita**
## WHAT
As title.

## WHY
Because we missed in the previous PR 👍

### CODE REVIEW

File: .github/workflows/release-provider.yml [hunk 1]
```diff
@@ -50,7 +50,7 @@ jobs:
         uses: crazy-max/ghaction-import-gpg@v3
         with:
           gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }}
-          passphase: ${{ secrets.PASSPHRASE }}
+          passphrase: ${{ secrets.PASSPHRASE }}
```
Comment (dragon3) [inline .github/workflows/release-provider.yml:53]: 😄

File: .github/workflows/release-provider.yml [hunk 2]
```diff
@@ -50,7 +50,7 @@ jobs:
...
       - name: Release provider
         uses: goreleaser/goreleaser-action@v1
```
Comment (dragon3) [inline .github/workflows/release-provider.yml:55]: In the master branch, we use `v2.4.1` 🤔 
```suggestion
        uses: goreleaser/goreleaser-action@v2.4.1
```

**Review by dragon3 — COMMENTED** [review]
Could you check the goreleaser-action version?
LGTM other than that 👍

File: .github/workflows/release-provider.yml (hunk 2 above)
Comment (KeisukeYamashita) [reply .github/workflows/release-provider.yml:55]: I think i missed another PR to cherry pick 🍒  🙇  thank you 🙇

**Review by dragon3 — APPROVED** [review]
Thank you for updating.
LGTM 👍

**Comment by KeisukeYamashita** [comment]
Thank you!!!! 🙇

**Pull request merged**